src/app/token-option-field-component-factories/token-option-field-component-factory.ts

```diff
@@ -209,7 +209,7 @@ export function createGradeThresholdComponentBuilder(
     return new FormFieldComponentBuilder()
         .setComp(createComponent(NumberInputFieldComponent, { environmentInjector: environmentInjector }))
         .editField((field) => {
-            field.label = 'Grade Threshold';
+            field.label = 'Passing Grade Threshold';
```

Comment:
This could be a little tricky since "Spend for Quiz Revision" token option also uses this component, but the grade threshold for that type of token option is a upper bound rather than a lower bound. Maybe we should make the label name as a parameter (and defaults to "Passing Grade Threshold").